programs/sbf/Cargo.lock

```diff
@@ -9386,7 +9386,7 @@ dependencies = [
  "serde_derive",
  "solana-accounts-db",
  "solana-bloom",
- "solana-bls",
+ "solana-bls-signatures",
```

Comment:
Is it normal that we are suddenly pulling a new crate here when change has nothing to do with it?

Reply:
This is leftover from a previous PR, we should have generated this as well.

Reply:
Ok just looks super sus to have cargo.lock changes for no apparent reason =)